modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/UnbindObjectDefinitionMVCResourceCommand.java

```diff
@@ -1,45 +0,0 @@
-/**
- * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
- * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
- */
-
-package com.liferay.object.web.internal.object.definitions.portlet.action;
-
-import com.liferay.object.constants.ObjectPortletKeys;
-import com.liferay.object.service.ObjectDefinitionLocalService;
-import com.liferay.portal.kernel.portlet.bridges.mvc.BaseMVCResourceCommand;
-import com.liferay.portal.kernel.portlet.bridges.mvc.MVCResourceCommand;
-import com.liferay.portal.kernel.util.ParamUtil;
-
-import javax.portlet.ResourceRequest;
-import javax.portlet.ResourceResponse;
-
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-
-/**
- * @author Feliphe Marinho
- */
-@Component(
-	property = {
-		"javax.portlet.name=" + ObjectPortletKeys.OBJECT_DEFINITIONS,
-		"mvc.command.name=/object_definitions/unbind_object_definition"
-	},
-	service = MVCResourceCommand.class
-)
-public class UnbindObjectDefinitionMVCResourceCommand
-	extends BaseMVCResourceCommand {
-
-	@Override
-	protected void doServeResource(
-			ResourceRequest resourceRequest, ResourceResponse resourceResponse)
-		throws Exception {
-
-		_objectDefinitionLocalService.unbindObjectDefinition(
-			ParamUtil.getLong(resourceRequest, "objectDefinitionId"));
```

Comment:
@victorKammerer one more thing, please remove this method as well `_objectDefinitionLocalService.unbindObjectDefinition` and its usages in tests we have to replace with [this](https://github.com/liferay/liferay-portal/blob/42b2af016e2d5b3e7df4d2019d73afdf0b2f235e/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java#L2494-L2499), after this change you can trigger the tests.

Thank you!

Reply:
Done it, @MarinhoFeliphe!